Emittery.d.ts

```diff
@@ -76,6 +76,16 @@ declare class Emittery {
 	 * specified.
 	 */
 	listenerCount(eventName?: string): number;
+
+	/**
+	 * Bind a `methodNames` or all `Emittery` methods if `methodNames` not defined into `target` object.
+	 */
+	bindMethods(target: Object, methodNames?: string[]): void;
+
+	/**
+	 * Returns a decorator which mixins `Emittery` as property `emitteryPropertyName` and `methodNames` or all `Emittery` methods if `methodNames` not defined into target class.
```

Comment:
```suggestion
	 * Returns a [decorator](https://github.com/tc39/proposal-decorators) which mixes in `Emittery` as a property named by `emitteryPropertyName` and `methodNames`, or all `Emittery` methods if `methodNames` is not defined, into target class.
```

Reply:
It does not conform to this proposal. (Hint: it is a proposal, it is not implemented anywhere yet.)